### ISSUE TITLE
[CAIA A11y Research]: Veteran-facing Forms, VA Form 21-4138

### ISSUE THREAD

**Issue by rachelshearerux** [department-of-veterans-affairs/va.gov-team#86767]
### Who's on your team?

```bash
Team name: Veteran Facing Forms
OCTO-DE product owner: Matthew Dingee
Product manager: Tricia Baker
Designer: Rachel Shearer
FE engineer: Jacob Penner, Robert Hasselle
Product/team Slack channel: #veteran-facing-forms
```

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Jeana Clark

### Is anyone on your team interested in becoming an accessibility champion?

Me! (Rachel Shearer :))

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

We'll schedule AT users between 7/18 - 7/26

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/21-4138/Product%20Outline.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1694025415991/ed022fe06bade5a04dfc62d6abf1eb249ac496c5?wid=0-1712174388644)
- [Prototype or mockups](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction)
- [Research plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Research%20Plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2024-03-reduce-4138-submissions/Conversation%20guide.md)
- [Research Epics](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/548)
- [Product Epic](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/548)
- [Collab cycle](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team/75601)

### Next steps

- [ ] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [X] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [X] Review our [assistive technology pilot guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) and request a pilot, if desired
- [ ] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
# CAIA A11Y Updates

## Study Dates Updated
The study dates for VFF VA Form 21-4138 have changed **from**: 7/18 - 7/26/2024, **to**: 7/29 - 8/9/2024. These changes are reflected in the following places.
- [x] Updated in this Research Ticket
- [x] Updated CAIA A11Y Calendar Holds
- [x] Updated CAIA A11Y Research Tracker

## Pilot Completed
- [x] Pilot Completed 7/18/2024 by @EvanAtCoforma 
- [x] Pilot Feedback - currently being implemented.

@NaomiPMC @EvanAtCoforma

**Comment by mutiaadhocteam** [comment]
**CAIA A11Y Findings**

Issue: "Continue" button is located at the end of the screen in VA Form 21-4138
Screen: All pages with "Continue" buttons
Version: All

<img width="960" alt="vetform2" src="https://github.com/user-attachments/assets/2e602d1d-d289-40f2-a666-374d3557b3e8">

Experience standard: [Findability](https://design.va.gov/about/experience-standards/#category-findability)

Description: The "Continue" button is located at the end of the screens throughout the form.

Details:

- Users may have difficulty determining the next steps for the form.
- Additionally, users may find it difficult to navigate through a considerable amount of content (form fields, dropdowns, text, etc.) before locating the "Continue" button.

Recommendation: The "Continue" button should be located next to the "Back" link.

Issue: No success message automatically announced upon updating information

<img width="960" alt="vetform1" src="https://github.com/user-attachments/assets/738a334d-987e-46b7-98cf-e771befdd0a0">

Screen: [6 of 6 Review application](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit)
Version: All

Experience standard: [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

Description: No success message is automatically announced upon updating information

Details:

- Users may have difficulty understanding that the information on the page has been successfully updated.
- Focus is sent back to the "Edit" button on updating information in a section.
- To determine that the information has been updated, the user would have to navigate through the content. Users may find this to be disruptive and cumbersome.

Recommendation: A success message should be automatically announced upon updating or otherwise editing information on the screen.

Issue: No error message automatically announced upon submitting an application containing errors
Screen: [6 of 6 Review application](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit)
Version: All

<img width="960" alt="vetform3" src="https://github.com/user-attachments/assets/620948f4-eb38-4373-82e6-1d802fa75db5">

Experience standard: [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)

Description: No error message is automatically announced upon submitting an application containing errors.

Details:

- Users may have difficulty understanding than an error is preventing them from submitting the form.
- Users will have to navigate through the content on the screen to understand that there is an error on the page.

Recommendation: An error message should be announced upon submitting a form with errors. This will inform the user that an error exists on the page and will prevent the user from unnecessary navigation of the page's content.

Issue: Focus is not automatically sent to the location of an error
Screen: [6 of 6 Review application](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit)
Version: All

Experience standard: [Findability](https://design.va.gov/about/experience-standards/#category-findability)

Description: Focus is not sent to the error that exists on the page upon submission of an application that contains an error.

Details:

- Users may have difficulty understanding where an error exists on the page.
- Users will have to navigate through the page's content to locate the error.

Recommendation: Focus should be sent to the error that exists on the page.
